templates/searchBooks.html

```diff
@@ -1,24 +1,91 @@
-<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@3.3.7/dist/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
-</head>
-<body>
-    <div class="container">
-        <div class="row">
-            <div class="col-md-12">
-                <h1 class="text-white" style="font-size:50px">Recommend Books</h1>
-                <form action="/recommend_books" method="post">
-                    <input name="user_input" type="text" class="form-control"><br>
-                    <input type="submit" class="btn btn-lg btn-warning">
-                </form>
-            </div>
+	<head>
+		<meta charset="UTF-8" />
+		<title>Book Recommender System</title>
+		<!-- Latest compiled and minified CSS -->
+		<link
+			rel="stylesheet"
+			href="https://cdn.jsdelivr.net/npm/bootstrap@3.3.7/dist/css/bootstrap.min.css"
+			integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u"
+			crossorigin="anonymous"
+		/>
+	</head>
+	<style>
+		.row {
+			margin-top: 40px;
+		}
+		.card {
+			height: 100%;
+			margin-bottom: 15px;
+		}
+		.card-body {
+			display: flex;
+			flex-direction: column;
+			justify-content: space-evenly;
+		}
+		.card-img-top {
+			max-height: 250px;
+		}
+		.card-title {
+			font-weight: 500;
+			font-size: 20px;
+			display: block;
+		}
+		.col-3 {
+			margin-bottom: 15px;
+		}
+		.card-text {
+			font-size: 14px;
+		}
+	</style>
+	<body>
+		<div class="container">
+			<div class="row">
+				<div class="col-md-12">
+					<h1 class="text-white" style="font-size: 50px">Recommend Books</h1>
+					<form class="form" action="/recommend_books" method="post">
+						<input
+							id="user_input"
+							name="user_input"
+							type="text"
+							class="form-control"
+						/><br />
+						<input
+							type="submit"
+							class="btn btn-lg btn-warning"
+							value="Submit"
+						/>
+					</form>
+				</div>
+			</div>
+			<div class="title">
+				<h2 class="recommendation-title"></h2>
+				{% if same_year_books_by_name %} {% for book in same_year_books_by_name
+				%}
+				<div class="col-md-3" style="margin-top: 50px">
+					<div class="card">
+						<div class="card-body">
+							<img class="card-img-top" src="{{book[2]}}" />
+							<p class="card-title">{{book[0]}}</p>
+							<p class="card-title">{{book[1]}}</p>
+						</div>
+					</div>
+				</div>
+				{% endfor %} {% endif %}
+			</div>
+		</div>
+	</body>
+	<script>
+		const form = document.getElementById("form");
+		form.addEventListener("submit", submitForm);
 
-        </div>
-    </div>
-
-</body>
-</html>
\ No newline at end of file
+		let user_input = document.querySelector("#user_input");
+		let divElement = document.querySelector(".title");
+		let title = document.querySelector(".recommendation-title");
+		title.textContent = `Books published in the same year`;
+		function submitForm() {
+			divElement.appendChild(title);
+		}
+	</script>
```

Comment:
As discussed, we can remove this script. No longer needed

Reply:
Hi Harshita, removed the js script.